### DISCUSSION TITLE
Search servers by region

### DISCUSSION THREAD

**Discussion by Naeris890** [FakeFishGames/Barotrauma#11914] [Ideas and suggestions]
Please allow players to search by region i get that you wont fix the ping displaying incorectly but can you please make server hosts select the region that their servers are hosted in as finding a server with ping below 250 is almost impossible for us australians

**Comment by Regalis11** [comment]
Thank you for the suggestion, but please do not post feature requests on the issue tracker! The issue tracker is only for bug reports.
